src/binaryen-c.cpp

```diff
@@ -1030,4 +1072,15 @@ void BinaryenSetAPITracing(int on) {
   }
 }
 
+//
+// ===== Emscripten support =====
+//
+
+#ifdef EMSCRIPTEN
+int __cxa_thread_atexit(void(*)(char*), char*, char*)
```

Comment:
Ok, I investigated that. Needed a little fix in emscripten, https://github.com/kripken/emscripten/pull/5261 . Nothing to block us here.